src/wordpress-playground/index.tsx

```diff
@@ -297,3 +299,104 @@ function zipNameToHumanName(zipName) {
 }
 
 main();
+
+async function overwriteFile() {
+	const targetFile = document.getElementById('target-file').value;
+	const targetContent = document.getElementById('target-content').value;
+	console.log(targetFile);
+	console.log(targetContent);
+	await workerThread.writeFile(targetFile, targetContent);
+}
+
+async function generateZip() {
+	await workerThread.run({
+		code: `
+			<?php
+				$zip = new ZipArchive;
+				$res = $zip->open('/wordpress-playground-export.zip', ZipArchive::CREATE);
+				if ($res === TRUE) {
+					$directories = array();
+					$directories[] = '/wordpress/';
+
+					while(sizeof($directories)) {
+						$dir = array_pop($directories);
+
+						if ($handle = opendir($dir)) {
+
+							while (false !== ($entry = readdir($handle))) {
+								
+								if ($entry == '.' || $entry == '..') {
+									continue;
+								}
+
+								$entry = $dir . $entry;
+
+								if (is_dir($entry)) {
+
+									$directory_path = $entry . '/';
+									array_push($directories, $directory_path);
+
+								} elseif (is_file($entry)) {
+
+									$zip->addFile($entry);
+								}
+							}
+							closedir($handle);
+						}
+					}
+					$zip->close();
+				}
+			?>
+		`,
+	});
+	const fileBuffer = await workerThread.readFileAsBuffer(
+		'/wordpress-playground-export.zip'
+	);
+	const file = new File([fileBuffer], 'wordpress-playground-export.zip');
+	saveAs(file);
+}
+
+async function importFile() {
+	const selectedFile = document.getElementById('file-input').files[0];
+	const fileContents = await selectedFile.arrayBuffer();
+	await workerThread.writeFile('/import.zip', new Uint8Array(fileContents));
+	await workerThread.run({
+		code: `
+			<?php
+				$zip = new ZipArchive;
+				$res = $zip->open('/import.zip');
+				if ($res === TRUE) {
+					$counter = 0;
+					while ($zip->statIndex($counter)) {
+						$file = $zip->statIndex($counter);
+						$fileString .= $file['name'] . ',';
+						if (
+							strpos($file['name'], 'wp-content/database') === false &&
```

Comment:
Would it make more sense to avoid putting it in the zip archive in the first place?